README.md

```diff
@@ -9,12 +9,12 @@ dive deeper into your code
 ## Installation
 
 ### Deps Project:
-1. In your `deps.edn`, make sure the `dev` alias is on the latest `repl-bootstrap`, which includes `hydrox` and `fudje`:
+1. In your `deps.edn`, make sure the `dev` alias is on the latest `repl-bootstrap`, which includes `hydrox` and `fudje`. Also add the old version of `core.match` so `hydrox` can properly parse html:
 ```clojure
   :dev      {:extra-deps
              {parkside-securities/repl-bootstrap {:git/url "git@github.com:parkside-securities/repl-bootstrap.git"
-                                                  :sha     "Anything later than e9de50b" }
-}
+                                                  :sha     "Anything later than e9de50b"}
+             org.clojure/core.match              {:mvn/version  "0.2.2"}}
```

Comment:
@mllehrer I might misunderstand your question in the stand-up. I'd like to put `core.match` to `repl-bootstrap` as the transitive dependency as our codebase doesn't have the direct dependency on it.

Reply:
Ah, ok. I will do that instead then.